distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column_Vector_Extensions.enso

```diff
@@ -0,0 +1,53 @@
+from Standard.Base import all
+
+import project.Data.Column.Column
+
+## Creates a column from a `Vector` of values.
+
+   Arguments:
+   - name: Name of the new column.
+Vector.to_column : Text -> Column
+Vector.to_column self name="Vector" =
+    Column.from_vector name self
+
+## Creates a column from a `Range` of values.
+
+   Arguments:
+   - name: Name of the new column.
+Range.to_column : Text -> Column
+Range.to_column self name="Range" =
+    Column.from_vector name self.to_vector
+
+## Creates a column from a `Date_Range` of values.
+
+   Arguments:
+   - name: Name of the new column.
+Date_Range.to_column : Text -> Column
+Date_Range.to_column self name="Date_Range" =
+    Column.from_vector name self.to_vector
+
+## Compute a single statistic on the column.
+
+   Arguments:
+   - statistic: Statistic to calculate.
+Column.compute : Statistic -> Any
+Column.compute self statistic=Statistic.Count =
+    self.compute_bulk [statistic] . first
+
+## Compute statistics on the column.
+
+   Arguments:
+   - statistics: Set of statistics to calculate.
+Column.compute_bulk : Vector Statistic -> Vector Any
+Column.compute_bulk self statistics=[Statistic.Count, Statistic.Sum] =
+    Statistic.compute_bulk self.to_vector statistics
+
+## Compute a single running statistic on the column.
+
+   Arguments:
+   - statistic: Statistic to calculate.
+   - name: Name of the new column.
+Column.running : Statistic -> Column
+Column.running self statistic=Statistic.Count name=statistic.to_text+" "+self.name =
+    data = Statistic.running self.to_vector statistic
+    Column.from_vector name data
```

Comment:
Is this searchable in the CB by typing `statistic`? If not then I think some alias may be needed.